### ISSUE TITLE
E-mail to "undisclosed-recipients"

### ISSUE THREAD

**Issue by Donnerstein** [YouPHPTube/YouPHPTube#1807]
E-mail functions

The e-mail for testing, and to confirm the mail address works ,. as well as the contact form. About sendmail, and also about smtp.
But, the notification of a user for their own subscribers, as well as the mail to all users and also the plugin: "Notifications" Do not work. An email will be sent with the recipient: "undisclosed-recipients".

with regards

**Comment by DanielnetoDotCom** [comment]
But the email arrives at the destination?

Notifications sent 100 emails at a time, that is why all destination emails are undisclosed

**Comment by Donnerstein** [comment]
Yes, some arrive, but not all. When I use Sendmail, these messages are already rejected by Sendmail. And an error is displayed on the page: "Could not execute: sendmail -t -i" Then in the Senmail log:

"... DSN undisclosed-recipiens:; ... list; syntax illegal for recipient addresses"

Sendmail works fine with other sites and php-mail.
I then get from Sendmail, the following report to the sender address, which I have entered on the Youphptube page as the sender.

`` `
The original message was received at Sun, 16 Jun 2019 13:25:20 +0200
from www-data @ localhost

   ----- The following addresses had permanent fatal errors -----
undisclosed-recipients :;

   ----- Transcript of session follows -----
553 5.1.3 undisclosed-recipients:; ... List :; syntax illegal for recipient addresses
`` `


I then tried it via an SMTP server, some messages are then sent, and then rejected by some providers. Because the syntax is not ok. The same message as Sendmail. I get the most messages back. I'm still building the youphptube, the site is not public yet. There is a total of only 25 members registered.
The test message as well as a single message to a user will be sent properly, via sendmail, and also via smtp. Only if it is to be sent to several addresses at the same time, or if the Notification plugin informs about a new event, the problem occurs.
